src/components/swimlanes/swimLanes.tsx

```diff
@@ -109,6 +114,10 @@ const SwimLanes = ({ lanesWithNumItems }) => {
           "& svg": {
             marginTop: "1px",
           },
+          color: "ui.blue",
+          _hover: {
+            textDecoration: "underline 1px dotted !important",
+          },
```

Comment:
For this block, some styles are needed. Replace lines 108-120 from  `sx={...}`  to the following:

```
__css={{
          display: { sm: "flex", md: "none" },
          fontWeight: "medium",
          justifyContent: "flex-end",
          marginTop: "s",
}}
```

Related to the issue above, it looks like the `sx` prop is not working as intended. Since there are specific styles that are required, use `__css` instead.

Reply:
yes, also, the `>` svg icon's fill color is black and not blue.... For some reason, the `Link` is breaking in this context and I can't tell why.